### ISSUE TITLE
Licensing and speech APIs (Dear Mark)

### ISSUE THREAD

**Issue by bitnom** [facebookresearch/seamless_communication#28]
I just want to put it on the record here that achieving anything close to what this model provides is prohibitively expensive and prone to technical issues, for any startup. We could really benefit from a conditional, commercial-friendly license for this model.

At least LLAMA2 gave us a pathway to bringing useful things to market. Open-source will eventually surpass this model anyway.  It seems like it would have been super-cool to lock startups into a  LLAMA2-like license, which would pay off down the road.

I got really excited when I saw the news headline for this model. Elevenlabs and bark are not fun to build around (Not throwing shade).

Please Mark, liberate us from them.

**Comment by jmp84** [comment]
Thank you for the feedback!

> Open-source will eventually surpass this model anyway.

We have no doubt and we hope that future models will surpass this m4t model.

Licensing is carefully considered for past, present and future releases.

**Comment by m-a-sch** [comment]
It is a fair point, but how does it sound when in the shoes of a consumer? "liberate us from them so that we can be captured by them"?

**Comment by upadrasta84** [comment]
I agree with this. Please make it a commercial license. LLaMa 2 is now everywhere. Hope this model can have similar success. without the commercial license, there is not much use of this model. I support @bitnom 's comments .

**Comment by bitnom** [comment]
> It is a fair point, but how does it sound when in the shoes of a consumer? "liberate us from them so that we can be captured by them"?

A fair point as well. It's not ideal but IMO it's better than what I would have expected before the proliferation of models of recent years began. It's certainly better than an API I can't modify, host, or a model that barely works.

The licensing situations we often find ourselves in now are bizarre. For instance: fine-tunes being released under creative commons non-commercial. That sort of behavior seems rather perverse to me. I think we could all benefit from some sort of weak copyleft but I digress.

**Comment by ejkitchen** [comment]
I'm disappointed you're releasing this with a non-commercial license. How are we supposed to use it? This approach just pushes me and several others, as above to prioritize open-source models. Many AI developers on Discord I've spoken to feel the same way. It's like you're pushing us away instead of inviting collaboration. These models lose long-term value if you alienate potential supporters. The whole idea of look how cool this is, but you really can't use it is absurd.

By the way, I fully support commercial licensing for large volumes, but small and mid-size companies should have royalty-free access. I completely understand you don't want Microsoft, OpenAI or others to use your stuff without royalties, but you should support small enterprises, especially those that are building apps for people who can't afford to run their own stuff and want a service that allows everyone to communicate seamlessly. What if someone wants to include translation as part of a much broader service, and their application is geared toward poorer countries or disenfranchised people? There are many scenarios where a business might want to use this commercially, not just for translation but as an extra feature to reach a much broader audience that cannot speak English or any language where economies are typically doing better than others.

This is a very bad decision. It feels like a slap in the face. Speaking to former Meta/Microsoft employees, some have left explicitly for this type of behaviour.

**Comment by zachNA2** [comment]
Since this has been re-opened are we going to see Meta re-evaluate the license for this model?

**Comment by zifeis** [comment]
curious about if/when it can be used commercially in the future

**Comment by BaohaoLiao** [comment]
any update for the reopen?
